docs/configuration/entrypoints.md

```diff
@@ -47,6 +50,9 @@ To redirect an entrypoint rewriting the URL.
     replacement = "http://mydomain/$1"
 ```
 
+!!! note
+    `regex` and `replacement` are not used during entrypoint redirection (if `entrypoint` is set in `redirect` structure).
```

Comment:
I would use the wording:

Please note, if you have `regex` and `replacement` used to rewrite a URL, ensure that you do not have `entrypoint` defined in the redirect structure, or they will be ignored (as noted above).